client/themes/default/components/page.vue

```diff
@@ -491,8 +502,8 @@ export default {
     this.$store.set('page/tags', this.tags)
     this.$store.set('page/title', this.title)
     this.$store.set('page/updatedAt', this.updatedAt)
-    if (this.commentsPermissions) {
-      this.$store.set('page/commentsPermissions', JSON.parse(atob(this.commentsPermissions)))
+    if (this.effectivePermissions) {
+      this.$store.set('page/effectivePermissions', JSON.parse(atob(this.effectivePermissions)))
```

Comment:
Even though it wouldn't break in this case, avoid using atob() since it doesn't handle non-latin characters. Instead use:
```js
JSON.parse(Buffer.from(this.effectivePermissions, 'base64').toString())
```